src/test/java/com/gildedrose/ItemsControllerTest.java

```diff
@@ -0,0 +1,58 @@
+package com.gildedrose;
+
+import com.gildedrose.goblins_grotto.Item;
+import com.gildedrose.items.Ageable;
+import com.gildedrose.items.DefaultItem;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import waiter.Request;
+import waiter.Response;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ItemsControllerTest {
+
+    ArrayList<Ageable> mockedDB;
+    ItemsController itemsController;
+
+    @BeforeEach
+    void setUp() {
+        mockedDB = new ArrayList<>();
+        itemsController = new ItemsController(new ItemsDataSourceMock<>(mockedDB));
+        mockedDB.add(new DefaultItem(new Item("foo", 3, 14)));
+
+    }
+
+    @Test
+    void returnsOkResponse_WhenSingleItemIsRetrieved() {
+        Request request = new Request("/inventory", Request.Method.GET.asString, "foo", "foo", "foo");
```

Comment:
What are the `foo`?

Reply:
The `foo` are the fields we don't care about for this test ie: protocol, request headers, request body